core/store/models/log_events.go

```diff
@@ -88,30 +88,43 @@ func TopicFiltersForRunLog(logTopics []common.Hash, jobID *ID) [][]common.Hash {
 func FilterQueryFactory(i Initiator, from *big.Int) (ethereum.FilterQuery, error) {
 	switch i.Type {
 	case InitiatorEthLog:
-		return newInitiatorFilterQuery(i, from, nil), nil
+		if from == nil {
+			from = i.InitiatorParams.FromBlock.ToInt()
+		} else if from != nil && i.InitiatorParams.FromBlock != nil {
+			from = utils.MaxBigs(from, i.InitiatorParams.FromBlock.ToInt())
+		}
+		q := ethereum.FilterQuery{
+			FromBlock: from,
+			ToBlock:   i.InitiatorParams.ToBlock.ToInt(),
+			Addresses: utils.WithoutZeroAddresses([]common.Address{i.Address}),
+			Topics:    i.Topics,
+		}
+		logger.Warnf("FilterQueryFactory %+v", q)
```

Comment:
Curious, why is this considered a Warning(..) as opposed to Info(..)?

Reply:
As far as I can see, this case isn't coverd by tests.  Anyway this could be improved?

Reply:
Agreed, ought to be, but wanted some input about the team's testing strategy before arriving at my own opinion.